eng/Versions.props

```diff
@@ -212,10 +212,12 @@
     <SystemIOFileSystemPrimitivesVersion>4.3.0</SystemIOFileSystemPrimitivesVersion>
     <SystemIOPipesAccessControlVersion>4.5.1</SystemIOPipesAccessControlVersion>
     <SystemIOPipelinesVersion>4.7.0</SystemIOPipelinesVersion>
+    <SystemManagementVersion>5.0.0-preview.8.20407.11</SystemManagementVersion>
```

Comment:
The version here is repeated and essentially lines up to the .NET SDK version we are using. Consider factoring this out to a property just as we do for our VS references. That will make future updates easier and more consistent.

Reply:
Should `MicrosoftCodeAnalysisNetAnalyzersVersion` be repurposed to do this?

https://github.com/dotnet/roslyn/blob/d2bc24adaaeffd8ef3755bc4ba9e2ce22e0bbb1b/eng/Versions.props#L31

Reply:
I think we should keep those separate. There have been times in the past where we have rev'd analyzers and SDK references at different times. Or another way to think about it is that we should expect drift between components that come from different GitHub repositories even if they all ship in the .NET SDK.

Reply:
Sounds good, I think we should look at doing this in a subsequent PR.